nixos/modules/services/databases/postgresql.nix

```diff
@@ -623,7 +623,46 @@ in
             TimeoutSec = 120;
 
             ExecStart = "${postgresql}/bin/postgres";
+
+            # Hardening
+            CapabilityBoundingSet = [ "" ];
+            DevicePolicy = "closed";
+            PrivateTmp = true;
+            ProtectHome = true;
+            ProtectSystem = "strict";
+            MemoryDenyWriteExecute = lib.mkDefault (config.services.postgresql.settings.jit == "off");
```

Comment:
This could be shortened to `cfg.settings.jit`

Reply:
AFAIK systemd only accepts yes/no whereas this is `on`/`off`.

Reply:
```
MemoryDenyWriteExecute = lib.mkDefault (cfg.settings.jit == "off");
```

is what I meant.